src/main/java/org/opensearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -109,20 +109,21 @@ protected void handleApiRequest(final RestChannel channel, final RestRequest req
 		try {
 			// validate additional settings, if any
 			AbstractConfigurationValidator validator = getValidator(request, request.content());
-			if (!validator.validate()) {
+			if (validator!= null && !validator.validate()) {
```

Comment:
If we change the condition to `validator == null || !validator.validate()` then the use of checks in the lines below is not required.

Reply:
This was added while testing and I forgot to remove it. Thanks @anijain-Amazon  for pointing this out. Will remove this check condition in next commit.